zulipterminal/ui.py

```diff
@@ -77,6 +77,26 @@ def set_footer_text(self, text_list: Optional[List[Any]]=None,
             time.sleep(duration)
             self.set_footer_text()
 
+    @asynch
+    def set_typeahead_footer(self, suggestions: List[Any],
+                             state: Optional[int],
+                             is_truncated: bool) -> None:
+        # wrap by space and use only name from tuple
+        suggestions = list(map(lambda x: ' ' + x[1] + ' ', suggestions))
+
+        if state is not None:
+            suggestions[state] = ('code', suggestions[state])
+
+        if is_truncated:
+            suggestions += [' [more] ']
+
+        if not suggestions:
+            suggestions = [' [No matches found]']
+        else:
+            suggestions.insert(0, [' '])  # Add leading space.
```

Comment:
It would be clearer if the logic above this conditional was in this else statement?

The leading space looks good!